modules/nested-elements/assets/js/index.js

```diff
@@ -0,0 +1,5 @@
+elementorCommon.elements.$window.on( 'elementor:pre-start', () => {
```

Comment:
Try to use `jQuery( window ).on( 'elementor:init', ... )`

Reply:
I want to give it a try to be the convention of loading compiled modules.

Using this syntax, the module can do things before editor loaded.